.github/workflows/ci.yml

```diff
@@ -27,6 +27,20 @@ jobs:
           --health-interval 10s
           --health-timeout 5s
           --health-retries 5
+      mysql:
+        image: mysql
+        ports:
+          - 8888:3306
```

Comment:
why not use 3306?

Reply:
I updated it. Looked like GH actions didn't permit using this port for Mysql Docker images if the default MySql process wasn't shutting down, but looks like it was changed too.